tests_integration/test_data_provider/test_databases/test_base.py

```diff
@@ -261,3 +261,75 @@ def test_load_pandas_dataframe_to_table_with_append(dst_dataset_fixture):
     assert rows[1] == (2,)
 
     dst_dp.drop_table(dataset)
+
+
+@pytest.mark.parametrize(
+    "dst_dataset_fixture",
+    [
+        {"name": "SqliteDataProvider"},
+        {
+            "name": "SnowflakeDataProvider",
+        },
+        {
+            "name": "BigqueryDataProvider",
+        },
+    ],
+    indirect=True,
+    ids=lambda db: db["name"],
+)
+def test_write_to_table_with_replace(dst_dataset_fixture):
```

Comment:
@sunank200 Now that we already have test cases for `load_pandas_dataframe_to_table` for `append` and `replace`. I think it would make more sense to just have a unit test case that only verifies that we are passing correct arguments to `load_pandas_dataframe_to_table` when calling `write`. WDYT?

Reply:
I think `load_pandas_dataframe_to_table` verifies loading using pandas dataframe but we have two paths in loading `source_ref` so this test is still useful.

Reply:
@sunank200 Make sense, so can we extend your tests to cover both paths? but for pandas, just the Unit test would be sufficient.

Reply:
@utkarsharma2 added the test here: https://github.com/astronomer/apache-airflow-provider-transfers/pull/45/commits/a11e1b13895dcc62e9e29bd7335ff426b5183d05